server/models/listItemModel.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
       } else {
         res.end();
       }
-    });
+    }, req.query.sort);
```

Comment:
Think it would be easier to read if this came after req.user.id. So we'd have req.user.id, req.query.sort, and then the callback. This would necessitate changing the order of args in each model as well.